.travis.yml

```diff
@@ -3,8 +3,8 @@ language: c
 
 env:
     global:
-    - BYOND_MAJOR="507"
-    - BYOND_MINOR="1286"
+    - BYOND_MAJOR="508"
```

Comment:
It would be nice if there was a compiletime error or warning outside of travis when compiling with pre-507 versions, since that throws a bad text error in malfunction.dm and people keep making issues about how the code doesn't compile while it does.